deploy/helm/dynatrace-oneagent-operator/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: dynatrace-oneagent-operator
+description: The Dynatrace OneAgent Operator Helm chart for Kubernetes and Openshift
+type: application
+version: 0.1.0
```

Comment:
While I presume both versions should be the same, perhaps we need to use 0.5.2 instead of 0.1.0 for example, on the actual released charts.